docs/traceability/index.md

```diff
@@ -853,6 +875,7 @@ output.traceability:
     responseHeader:
       show: ${TRACEABILITY_REDACTION_RESPONSEHEADER_SHOW:[]}
       sanitize: ${TRACEABILITY_REDACTION_RESPONSEHEADER_SANITIZE:[]}
+  maskingCharacters: ${TRACEABILITY_REDACTION_MASKING_CHARACTERS:"\u007B*\u007D"} # unicode for {*}
```

Comment:
Did the following not work?  To prevent the need of double quotes in env vars?

`maskingCharacters: "${TRACEABILITY_REDACTION_MASKING_CHARACTERS:\u007B*\u007D}" # unicode for {*}`